WordPress/src/main/res/layout/theme_activation_bottom_sheet.xml

```diff
@@ -0,0 +1,162 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <include
+        android:id="@+id/bottom_sheet_handle"
+        layout="@layout/bottom_sheet_handle_view"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content" />
+
+    <ImageButton
+        android:id="@+id/close_button"
+        android:layout_width="@dimen/theme_bottom_sheet_close_button_size"
+        android:layout_height="@dimen/theme_bottom_sheet_close_button_size"
+        android:layout_alignParentEnd="true"
+        android:layout_below="@id/bottom_sheet_handle"
+        android:layout_margin="@dimen/margin_extra_large"
+        android:background="@drawable/close_action_sheet_circle_selector"
+        android:contentDescription="@string/label_close_button"
+        android:cropToPadding="true"
+        android:padding="@dimen/theme_bottom_sheet_close_button_padding"
+        android:src="@drawable/ic_close_black_24dp"
+        android:tint="@color/theme_action_sheet_close_button_tint" />
+
+    <TextView
+        android:id="@+id/theme_name_tv"
+        style="@style/ThemeActionSheetTitle"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_below="@id/close_button"
+        android:layout_centerHorizontal="true"
+        android:text="Theme Name" />
+
+    <TextView
+        android:id="@+id/info_tv"
```

Comment:
Hey @kashandanish 👋 
I just noticed a pretty minor inconsistency with the rest of the project involving layout naming. Adding the layout type as a suffix to the view names (`_tv`, `_ll`, `_iv`) looks like a good convention but I wasn't able to find any other occurrence of this practice in the project.

Reply:
Thank you for making this change @kashandanish 👍